edge/hoymilesapi.py

```diff
@@ -260,8 +262,9 @@ def adjust_solar_data(self, solar_data: dict) -> dict:
         solar_data['real_power_measurement'] = str(real_power)
         solar_data['real_power_total_increasing'] = str(real_power)
         solar_data['power_ratio'] = str(power_ratio)
-        solar_data['capacitor'] = str(capacidade)
-        solar_data['capacitor_kW'] = str(capacidade / 1000)
+        solar_data['array_size'] = str(capacidade)
+        solar_data['array_size_kW'] = str(capacidade / 1000)
```

Comment:
So if you are editing this file, could you also rename 'caoacidade' variable, please? Some of the code is still not translated by me to English.

Reply:
I've made the changes.  I would be fine with using capacity (capacidade) everywhere, but using capacitor was confusing for me.  Would you like me to translate all the comments in the code to English?  I don't see a need to do so.

Reply:
If you have time, please ;) To be honest, I found another way to collect data from my inverters, so no I have less time to develop this addon.

Reply:
I will start a new branch to change all the comments from Portuguese to English.  Can this pull request be closed, or is there something more that I need to do?